src/it/ajc/verify.groovy

```diff
@@ -30,3 +30,4 @@
 
 assert !new File(basedir, 'target/classes/jcabi-ajc.log').exists()
 assert new File(basedir, 'target/jcabi-ajc.log').exists()
+assert new File(basedir, 'target/unwoven/Sample.class').exists()
```

Comment:
@amihaiemil can we somehow verify that woven classes are where they were before?